### DISCUSSION TITLE
Is it possible to disable line wrapping?

### DISCUSSION THREAD

**Discussion by jackkoenig** [outr/scribe#533] [Q&A] [answered]
Basically the question in the title. I have a project using scribe and I'd like for the things I log that don't contain newlines to be printed without newlines (for parsing purposes). Is it possible to disable the automatic line wrapping somehow?

Thanks!

**Comment by darkfrog26** [comment, marked as answer]
@jackkoenig absolutely. That's just the default formatter. When you configure your logging you can either create your own formatter or use one of the existing options. The default is 'advanced': https://github.com/outr/scribe/blob/master/core/shared/src/main/scala/scribe/format/Formatter.scala#L51, but if you don't want multi-line I'd recommend using `enhanced`:
```scala
Logger
  .root
  .clearHandlers()
  .withHandler(minimumLevel = Some(Level.Info), formatter = Formatter.enhanced)
  .replace()
```

**Reply by jackkoenig** [reply]
Thank you @darkfrog26! Works great and thank you for the explanation.
